source/_components/climate.toon.markdown

```diff
@@ -1,8 +1,8 @@
 ---
 layout: page
-title: "Toon"
-description: "Instructions how to integrate Toon Climate devices within Home Assistant."
-date: 2017-10-21 12:00
+title: "Toon Thermostat"
+description: "Instructions how to integrate Toon Thermostats within Home Assistant."
```

Comment:
`Thermostats` is not part of the title or product name and should be written lower caps.